src/services/etherspot.js

```diff
@@ -124,8 +118,27 @@ class EtherspotService {
     this.subscription.unsubscribe();
   }
 
-  getAccount(accountAddress: string): Promise<?EtherspotAccount> {
-    return this.sdk.getAccount({ address: accountAddress }).catch((error) => {
+  getSdkForChain(chain: Chain): ?EtherspotSdk {
+    const network = networkNameFromChain(chain);
+    if (!network) {
+      reportErrorLog('EtherspotService getSdkForChain failed', { chain });
+      return undefined;
```

Comment:
Maybe just return? And for the one below.

```suggestion
      return;
```